tests/Smoke/CiIntegrationTest.php

```diff
@@ -251,6 +251,9 @@ private static function executeCommand($command)
 
     private static function executeScript(array $scriptParts)
     {
-        return BashScriptExecutor::create($scriptParts, self::$fixtureDir)->getResult();
+        // @TODO: drop $scriptInit, for now it's needed, as defaut `set -eu` is causing our scripts to crash
+        $scriptInit = array('#!/bin/sh', 'set -e', '');
```

Comment:
@EvgenyOrekhov, I remember that you wanted to fix `s/e/eu` globally.
during that, could you handle this one as well ?

Reply:
I created #3654 for `set -eu` and referenced this PR there.